lib/util.js

```diff
@@ -119,12 +119,14 @@ const util = {
       parseMatch();
     }
 
-    return ({
+    const result = {
       command: command,
       match: match,
       matchArgs: matchArgs,
       pipes: pipes
-    });
```

Comment:
If I had understood correctly (please confirm) command prop contains the raw command string, match contains the eventually found Command object; matchArgs is an array of matched arguments, and pipes is an array of subsequent command in a pipe.